x-pack/plugins/security/server/authentication/api_keys/api_keys.test.ts

```diff
@@ -407,6 +408,43 @@ describe('API Keys', () => {
     });
   });
 
+  describe('validate()', () => {
+    it('returns false when security feature is disabled', async () => {
+      mockLicense.isEnabled.mockReturnValue(false);
+      const result = await apiKeys.validate({
+        id: '123',
+        api_key: 'abc123',
+      });
+      expect(result).toBeFalsy();
+      expect(mockClusterClient.asScoped).not.toHaveBeenCalled();
+    });
+
+    it('calls callCluster with proper parameters', async () => {
+      mockLicense.isEnabled.mockReturnValue(true);
+      mockScopedClusterClient.asCurrentUser.security.invalidateApiKey.mockResponseOnce({
+        invalidated_api_keys: ['api-key-id-1'],
+        previously_invalidated_api_keys: [],
+        error_count: 0,
+        error_details: [],
+      });
```

Comment:
__nit:__ looks like a copy-paste error, we don't need to mock `invalidateApiKey` here.
```suggestion
```